src/PrestaShopBundle/Resources/config/services/adapter/customer_message.yml

```diff
@@ -2,8 +2,8 @@ services:
   _defaults:
     public: true
 
-  prestashop.adapter.customer_message.command_handler.add_order_customer_message:
-    class: PrestaShop\PrestaShop\Adapter\CustomerService\CommandHandler\AddOrderCustomerMessageHandler
+  prestashop.core.domain.customer_service.command_handler.add_order_customer_message:
```

Comment:
Actually, this one (along with all the other moved handlers) should now be placed into https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/config/services/core/domain/customer_service.yml